README.md

```diff
@@ -39,8 +39,10 @@ This module provides you with a convenient `Keptn` helper struct that allows you
 Keptn event. The helper struct can be initialized by passing a CloudEvent to the `NewKeptn` function. Example:
 
 ```go
-func HandleEvent(event cloudevents.Event) error {
-	keptnHandler, err := keptn.NewKeptn(&event, keptn.KeptnOpts{})
+func HandleEvent(ctx context.Context, event cloudevents.Event) error {
+    keptnHandler, err := keptn.NewKeptn(&event, keptn.KeptnOpts{
+		Context: ctx,
+	})
 	if err != nil {
 		return nil, err
 	}
```

Comment:
few lines below here there is a call to `keptnHandler.SendCloudEvent`. Please use the function you introduced `SendEventWithContext`. Furthermore, please also update other function names to their `*WithContext()` version.

Reply:
The `Keptn` struct has a new property `Context` which is passed on creation. The `SendCloudEvent` method on it uses this context, and internally it is changed now to call the `.Send(ctx...)` so there's no need for a `SendEventWithContext`. That makes existing code work "as-is" and to leverage propagation they just need to provide the context to the keptn struct instance. This is one of the approaches defined in this article: https://go.dev/blog/context-and-structs

Apart from it, in this readme I already changed the ocurrences to call the new `*WithContext()` (It was just the example to get the events)